catkin_home/src/main_engine/src/action_manager.py

```diff
@@ -57,8 +61,9 @@ def new_action_request_callback(self, msg):
         print("args: ")
         print(msg.args)
         # registered action
-        new_action = Action(msg.intent,
-                            msg.action_client_binded, msg.args)
+        new_action = self.createAction(msg)
+        if(new_action == None):
+            print("No action registered with that name" + msg.intent)
```

Comment:
Is there any default action if this statement is true? Or should it just display the message?

Reply:
It should actually throw an exception, but for now a message is fine i think.